fast64_internal/utility.py

```diff
@@ -523,6 +531,115 @@ def deleteIfFound(filePath, stringValue):
 			fileData.write(stringData)
 		fileData.close()
 
+def yield_children(obj: bpy.types.Object):
+	yield obj
+	if obj.children:
+		for o in obj.children:
+			yield from yield_children(o)
+
+def store_original_mtx():
+	active_obj = bpy.context.view_layer.objects.active
+	for obj in yield_children(active_obj):
+		obj['original_mtx'] = obj.matrix_local
+
+def rotate_bounds(bounds, mtx: mathutils.Matrix):
+	return [
+		(mathutils.Vector(b) @ mtx).to_tuple()
```

Comment:
Are you aware this means the rotation is inverted? `v @ mtx` is the same as `mtx.transposed() @ v`, and transpose on a rotation is the same as inverting it
If you're only using this to ditch the translation part you should use `mtx.decompose()` similarly to your `translation_rotation_from_mtx` function

Reply:
using `translation_rotation_from_mtx` isn't working here, but there shouldn't be any translation in the matrix anyways. i can change it to `mtx.transposed() @ v` if you think its clearer, i just don't understand exactly why that produces the same output in this scenario